cmd/lncli/commands.go

```diff
@@ -2387,12 +2387,26 @@ func updateChannelPolicy(ctx *cli.Context) error {
 		return errors.New("inbound_fee_rate_ppm out of range")
 	}
 
+	// Inbound fees are optional. However, if an update is required,
+	// both the base fee and the fee rate must be provided.
+	var inboundFee *lnrpc.InboundFee
+	switch {
+	case ctx.IsSet("inbound_base_fee_msat") !=
+		ctx.IsSet("inbound_fee_rate_ppm"):
+		return errors.New("both parameters must be provided: " +
+			"inbound_base_fee_msat and inbound_fee_rate_ppm")
+
+	case ctx.IsSet("inbound_fee_rate_ppm"):
+		inboundFee = &lnrpc.InboundFee{
+			BaseFeeMsat: int32(inboundBaseFeeMsat),
+			FeeRatePpm:  int32(inboundFeeRatePpm)}
```

Comment:
Close bracket should be on a new line.

Reply:
I see how using the sub-proto here helps us check for nil-ness. But don't we need to use the approach for the individual fields? Otherwise if the the base fee value is set, but the ppm value isn't, then the msat value will get reset to zero.

Reply:
In the future we could deprecate the other fields and use this method for them as well, as it seems to lead to a non-intuitive way to update values where one has to echo the non-updated values.

Reply:
We could make `BaseFeeMsat` and `FeeRatePpm` using the `optional` flag in proto3 and check their nil-ness - we need to update `falafel` first tho, as last time I tried to use it, `make rpc` would fail with,
```
walletkit.proto: is a proto3 file that contains optional fields, but code generator protoc-gen-custom hasn't been updated to support optional fields in proto3. Please ask the owner of this code generator to support proto3 optional.--custom_out:
```

Reply:
My primary goal was to ensure backwards compatibility with older RPC clients, so that a PolicyUpdate from clients that do not recognize inbound fees does not overwrite the InboundFees with 0. Many noderunners use several clients. However, clients that use InboundFees must set both fields.  

But I would also prefer an optional flag. Then you could also do without the submessage, which is more of a workaround. 

Who can customize Falafel, if optionality is desired?